pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+{
+  "acl": {
+    "create": {
+      "examples": [{
+        "cmd": "fastly acl create --name robots --version latest",
+        "title": "Create a new ACL attached to the specified service version"
```

Comment:
Wherever the example uses `--version latest`, the title should say something like "the highest numbered existing service version".  You could append "If the latest service version is not editable, this command will error", or could put that in a description - or leave it out